src/microbe_stage/editor/upgrades/VacuoleUpgradeGUI.cs

```diff
@@ -37,8 +37,17 @@ public override void _Ready()
 
     public void OnStartFor(OrganelleTemplate organelle, GameProperties currentGame)
     {
-        shownChoices = SimulationParameters.Instance.GetAllCompounds().Values
-            .Where(c => !c.IsEnvironmental && !c.IsAgent).ToList();
+        // Probably can be replaced with better logic
```

Comment:
Yeah, the upgrade GUI should have a private variable that holds the result of `SimulationParameters.Instance.GetCompound("mucilage")`. then the internal name of that variable can be compared with the thing that should be mucilage.

You can take a look at how for example the cell editor uses the sunlight variable:

https://github.com/Revolutionary-Games/Thrive/blob/master/src/microbe_stage/editor/CellEditorComponent.cs#L234